contracts/common/mixin/RootChainValidator.sol

```diff
@@ -112,20 +95,20 @@ contract RootChainValidator is RootChainable, Lockable {
   // validate tx
   function validateTx(
```

Comment:
if func is calling `validateTxReceiptExistence`. If this func is supposed to do both, then let's call it something else.